build/devbase/godeps.sh

```diff
@@ -32,6 +32,7 @@ ${GOPATH}/bin/glock sync github.com/cockroachdb/cockroach
 pkgs="
 github.com/biogo/store/interval
 github.com/biogo/store/llrb
+github.com/cenkalti/backoff
```

Comment:
Ah yes. We should probably hook this into `go generate`/`make check` since we keep forgetting it. I'd still prefer to implement this ourselves, though, even if the dependency is just one small library.